src/index.js

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Let’s not disable linter rules these are here to help us write a good price of code